packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
```

Comment:
again this is propName which is leading the the confusion below, the thing above is the className unless it is a boolean